src/vs/workbench/contrib/codeEditor/browser/suggestEnabledInput/suggestEnabledInput.ts

```diff
@@ -145,10 +145,10 @@ export class SuggestEnabledInput extends Widget implements IThemable {
 			if (options.focusContextKey) { options.focusContextKey.set(true); }
 			addClass(this.stylingContainer, 'synthetic-focus');
 		})));
-		this._register((this.inputWidget.onDidBlurEditorText(() => {
-			if (options.focusContextKey) { options.focusContextKey.set(false); }
-			removeClass(this.stylingContainer, 'synthetic-focus');
-		})));
+		// this._register((this.inputWidget.onDidBlurEditorText(() => {
```

Comment:
Is this removing intentional? if so why?

Reply:
That was supposed to be re-enabled... I'll send out an update. Thanks Udeesha.